prompts/sources.go

```diff
@@ -657,3 +683,75 @@ func configureRegistry(source *workflow.Source, orgSlug, workspaceSlug, sourceNa
 	source.Registry = registryEntry
 	return nil
 }
+
+type FormatType string
+
+const (
+	FormatJSON FormatType = "json"
+	FormatYAML FormatType = "yaml"
+)
+
+type blueprintRequest struct {
+	ID string `json:"id"`
+}
+
+type blueprintResponse struct {
+	ID        string     `json:"id"`
+	Spec      string     `json:"spec"`
+	CreatedAt time.Time  `json:"created_at"`
+	Format    FormatType `json:"format"`
+}
+
+var (
+	ErrMsgFailedToFetchBlueprint  = errors.New("failed to fetch sandbox session")
+	ErrMsgFailedToSaveBlueprint   = errors.New("failed to save sandbox session")
+	ErrMsgFailedToDecodeBlueprint = errors.New("failed to decode sandbox session")
+)
+
+func fetchAndSaveBlueprint(ctx context.Context, blueprintID string) (string, error) {
+	baseURL := "https://api.speakeasy.com"
+	url := fmt.Sprintf("%s/v1/schema_store", baseURL)
+
+	var reqBody blueprintRequest
+	reqBody.ID = blueprintID
+	body, err := json.Marshal(reqBody)
+	if err != nil {
+		return "", ErrMsgFailedToDecodeBlueprint
+	}
+	req, err := http.NewRequest(http.MethodGet, url, bytes.NewBuffer(body))
+	if err != nil {
+		return "", ErrMsgFailedToFetchBlueprint
```

Comment:
I'm not a SDK purist but we do have an SDK which could do this, just saying 🤣

Reply:
We don't use that in the CLI I don't think!

Reply:
Yep we do! go-client-SDK from memory

Reply:
Ah you mean: 


```
speakeasyclientsdkgo "github.com/speakeasy-api/speakeasy-client-sdk-go/v3"
```

Yeah I don't really have much of an opinion

Reply:
It's a very simple endpoint 😆

Reply:
Replaced it now - easier 👍🏻